tests/mt/test_Problem1D_againstAnalyticHalfspace.py

```diff
@@ -1,162 +0,0 @@
-import unittest
```

Comment:
is this a duplicate / redundant test? (just making sure that deleting it is intentional)